projects/packages/sync/src/modules/class-woocommerce-hpos-orders.php

```diff
@@ -160,19 +160,24 @@ public function get_object_by_id( $object_type, $id ) {
 	 * @return array
 	 */
 	public function get_objects_by_id( $object_type, $ids ) {
-		if ( 'order' !== $object_type ) {
-			return $ids;
+		if ( 'order' !== $object_type || empty( $ids ) || ! is_array( $ids ) ) {
+			return array();
 		}
-		$orders      = wc_get_orders(
+
+		$orders = wc_get_orders(
```

Comment:
@vedanshujain Added you as a reviewer mostly to confirm this is the recommended way to ensure we fetch all the orders, including the ones with `trash` and `auto-draft` statuses via `wc_get_orders`